caffeine@patapon.info/extension.js

```diff
@@ -228,11 +226,18 @@ class CaffeineToggle extends QuickSettings.QuickMenuToggle {
             this._itemsSection.addMenuItem(item);
         }
         this.menuEnabled = TIMERS.length > 2;
+     
+        // Select active duration
```

Comment:
What's the logic here ? Reset but not always ? :)

Reply:
I need to call `_sync()` at this step, this function is also called when `TIMER_KEY` changed. I wanted to avoid calling two time `_sync()` in case we reset timer to 0.

Reply:
It reset only when duration range changes in settings, but not when the extension starts. So this way, it keep the last option selected when Caffeine restart.